src/components/Patient/ManagePatients.tsx

```diff
@@ -638,7 +638,26 @@ export const PatientManager = () => {
                         startIcon="l-clock"
                         text="Review Missed"
                       />
+                    )} */}
+                  {patient.review_time &&
+                    !patient.last_consultation?.discharge_date &&
+                    Number(patient.last_consultation?.review_interval) > 0 && (
+                      <Chip
```

Comment:
Let's switch to using Shadcn's [Badge](https://ui.shadcn.com/docs/components/badge) here. You can port over the styles from Chip.

Reply:
ok